app/controllers/concerns/idv/proofing_concern.rb

```diff
@@ -0,0 +1,55 @@
+module Idv
+  module ProofingConcern
```

Comment:
Went over this while pairing with Sonia. I wasn't sure why this and the VerifyInfoConcern were split up. It turns out it is there to make rubocop happy. I [opened a PR](https://github.com/18F/identity-idp/pull/7880) to propose removing that rule.

In the meantime I proposed that Sonia add a comment to disable the linter here so we can keep these together.